requirements/dev.txt

```diff
@@ -104,7 +105,7 @@ pyparsing==2.4.7          # via -r requirements/doc.txt, -r requirements/test-ma
 pytest-catchlog==1.2.2    # via -r requirements/test.txt
 pytest-cov==2.10.0        # via -r requirements/test.txt
 pytest-django==3.9.0      # via -r requirements/test.txt
-pytest==5.4.3             # via -r requirements/test.txt, pytest-catchlog, pytest-cov, pytest-django
```

Comment:
ya this is bound to cause failures, unless someone has resolved issues

I would pull it out of the files manually ... maybe we can just constrain pytest somewhere, in constraints.txt ?

Reply:
this thread kind of died somewhere : https://edx-internal.slack.com/archives/CD89W7Z7F/p1596055879160500

Reply:
ok, constraining pytest to 5.4.3 as per master version

Reply:
hmm not sure where you're constraining it but since this is an automatically generated PR should we just close it, assuming another one would exist?